bin/data_frame_analyzer/Main.cc

```diff
@@ -140,8 +149,46 @@ int main(int argc, char** argv) {
         return EXIT_FAILURE;
     }
 
-    auto analysisSpecification =
-        std::make_unique<ml::api::CDataFrameAnalysisSpecification>(analysisSpecificationJson);
+    auto resultsStreamSupplier = [&ioMgr]() {
+        return std::make_unique<ml::core::CJsonOutputStreamWrapper>(ioMgr.outputStream());
+    };
+
+    // TODO Factor out these constants to an extra header file, e.g api/ElasticsearchStateIndex.h
+    const std::string ML_STATE_INDEX(".ml-state");
+    const std::string REGRESSION_TRAIN_STATE_TYPE("model_state");
```

Comment:
I think this has to be something different from `model_state` say `predictive_model_train_state`.

Reply:
Done.